src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -9,6 +9,7 @@ import execa from 'execa'
 import fs from 'fs'
 import net from 'net'
 import pRetry from 'p-retry'
+import fetch from 'cross-fetch'
```

Comment:
This had to be added as `new URL`(on L221) fails as the `BrowserEngine` was removed. I'm pretty sure this could also be achieved by adding "DOM" to tsconfig.